libwebauthn/src/transport/cable/tunnel.rs

```diff
@@ -513,7 +555,72 @@ async fn connection_recv_initial(
     Ok(initial_message.info.to_vec())
 }
 
+async fn connection_recv_update(message: &[u8]) -> Result<Option<CableLinkingInfo>, Error> {
```

Comment:
It's more or less equivalent, but I think we could use a custom deserialize-implementation for this, like done here: https://github.com/mozilla/authenticator-rs/blob/ctap2-2021/src/ctap2/commands/get_assertion.rs#L734

Reply:
Agreed, I think we'll have to do this more widely to fix #66 very soon.

Reply:
Sure